testing/test_argcomplete.py

```diff
@@ -91,14 +91,18 @@ def __call__(self, prefix, **kwargs):
 
 class TestArgComplete(object):
     @pytest.mark.skipif("sys.platform in ('win32', 'darwin')")
-    def test_compare_with_compgen(self):
+    def test_compare_with_compgen(self, monkeypatch):
         from _pytest._argcomplete import FastFilesCompleter
 
         ffc = FastFilesCompleter()
         fc = FilesCompleter()
-        for x in ["/", "/d", "/data", "qqq", ""]:
```

Comment:
based on the content i believe the test can be altered to generally run it in a folder, i would like to propose using the the tmpdir instead of home - since it prevents unnecessary data leaks in failures/errors

Reply:
also great dob in discovering a relatively simple fix

Reply:
Done.